VelodyneHDL/IO/Lidar/Velodyne/vtkVelodynePacketInterpreter.cxx

```diff
@@ -867,7 +867,7 @@ void vtkVelodynePacketInterpreter::ProcessPacket(unsigned char const * nextData,
         }
 
         // Sanity check (last 4 bits of confidenceValues[j] should always be empty)
-        if (confidenceValues.at(laserID) & 0x000f != 0)
+        if (confidenceValues.at(laserID) & 0x000f)
```

Comment:
Order of operations puts `!=` ahead of  `&`, so this change actually changes the behavior, but actually implements the intent implied by the comment above.